### ISSUE TITLE
Images shown in wrong order

### ISSUE THREAD

**Issue by Dreamfish19** [stevenwanderski/bxslider-4#899]
Hi,

I've implemented the latest v4.2.5 on my site due to other problems.

But since then the images appear in all browsers in wrong order.

The affected site is:

http://www.dr-grabowski.de/

I already tried preloading all images, and also 

.bx-clone {
display: none;
}

but this does not help.

Any other suggestions?

**Comment by eheiser** [comment]
I have a similar issue - did you get this resolved?

**Comment by Dreamfish19** [comment]
Yes, I went back to version 4.1.2 which is the latest rock stable version to me ...

**Comment by vicky9971** [comment]
@eheiser @Dreamfish19 
This issue can be solved by modifying the "setSlidePosition" method as : 
var setSlidePosition = function(){
            setTimeout(function(){
               //All the code from setSlidePosition() goes here..
            }, 30);
        }

In this plugin, the DOM is updated dynamically and position() being a jQuery function evaluates the position of the slides relative to the document. Since, no such content exist in DOM initially, it returns (0,0) whereas it should return the amount by which the slider needs to be rotated so as to place the correct slide in place.
So, I have added a timeout of 30ms (it can be even less) which allows the plugin to make changes to DOM thus the position() gives the right amount of displacement required in order to put the correct slide in place.
